h2o-automl/src/main/java/ai/h2o/automl/AutoML.java

```diff
@@ -1014,7 +1014,7 @@ void defaultGBMs(Key<Grid> gridKey) {
     StackedEnsembleModel.StackedEnsembleParameters stackedEnsembleParameters = new StackedEnsembleModel.StackedEnsembleParameters();
     stackedEnsembleParameters._base_models = allModelKeys.toArray(new Key[0]);
     stackedEnsembleParameters._valid = (getValidationFrame() == null ? null : getValidationFrame()._key);
-    stackedEnsembleParameters._keep_levelone_frame = true;
+    stackedEnsembleParameters._keep_levelone_frame = true; //TODO Why is this true? Can be optionally turned off
```

Comment:
Regarding your TODO comment: I think the reason this is true is that it fixed a leaked key issue.